### DISCUSSION TITLE
warn

### DISCUSSION THREAD

**Discussion by TahhanCoding** [Flinesoft/Prayer#35] [General]
Hi, I have just seen your app and I have full respect of your work.
But, You must know that prayers can't be done using another language except Arabic. And, can't be done using headphones either. Although you can read English translation to understand what you are saying, but you must read in Arabic during your prayer.

**Comment by Jeehut** [comment]
Hi @Ahmed0Shaban, thank you for taking the time to start the first discussion here!

Unfortunately, you are making bold statements here without any proof. Also, I have put an answer to your concerns into the app, just open the FAQ section and you'll see how I see it.

Please note that the Quran is the only source I care about, because any other source is just a failable humans opinion, sometimes million times rehashed, but still failable and not to be trusted. Some people believe the prophet was some kind of non-human being and was unfailable and thus everything he did is correct, even though the Quran itself says otherwise. Only God is unfailable and if someone believed otherwise, that would be nothing else but stating there's another God (the prophet, in this case, which many unfortunately believe in). Also, even if the prophet was unfailable, his world view was limited and it's impossible he could have foreseen our times today and how we should be living with all the technology and globalized world that we're in. So what people write about the prophet doesn't matter as much to me. What really matters is the written word of God.

This is why we should refer to the only unfailable source, which is the Quran, which my app helps you read during your regular prayers, which is what the prayers were invented for: To remind yourself of Gods words and guidance. How can you be guided in a language you don't speak? That doesn't make sense to me.

But if you want to do your prayers with just the app open for reading the translation while you are saying the words in Arabic in your head, you're free to do so. Just don't state that this is the _only_ way to do your prayers. How do you know? Only God knows. And to me what you're saying doesn't make sense at all. Why would the language matter? Only the meaning matters! And if you speak a language you don't understand, there's 0% meaning left. How can people prefer this over their native language? I don't understand.

**Comment by Jeehut** [comment]
I'm going to lock this discussion as I don't believe a fruitful discussion is possible based on how unobjective the discussion was started. The title also isn't anywhere near being clear what the discussion is supposed to be about.

If you have any concerns about the purpose of the app, please create a new discussion with a clear topic and provide sources for any bold statements you make. Also, make sure to read the built-in FAQ in the app before starting any discussions, I've answered the most frequently asked questions there already.
